test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -109,283 +109,245 @@ func (s *snapshottableTestSuite) DefineTests(driver TestDriver, pattern testpatt
+				case testpatterns.PreprovisionedCreatedSnapshot:
+					createPreprovisionedSnapshot()
+				default:
+					framework.Failf("%s is not a valid snapshot type", pattern.SnapshotType)
+				}
+			})
+
+			ginkgo.It("should remove the volume snapshot content with delete policy", func() {
+				DeleteAndWaitSnapshot(dc, vs.GetNamespace(), vs.GetName(), framework.Poll, framework.SnapshotDeleteTimeout)
+				ginkgo.By("checking the SnapshotContent has been deleted")
+				err = utils.WaitForGVRDeletion(dc, SnapshotContentGVR, vscontent.GetName(), framework.Poll, framework.SnapshotDeleteTimeout)
+				framework.ExpectNoError(err)
+			})
+
+			ginkgo.It("should not remove the volume snapshot content with retain policy", func() {
+				vsc.Object["deletionPolicy"] = "Retain"
+				vsc, err = dc.Resource(SnapshotClassGVR).Update(context.TODO(), vsc, metav1.UpdateOptions{})
+
+				vscontent.Object["spec"].(map[string]interface{})["deletionPolicy"] = "Retain"
+				vscontent, err = dc.Resource(SnapshotContentGVR).Namespace(vscontent.GetNamespace()).Update(context.TODO(), vscontent, metav1.UpdateOptions{})
+				framework.ExpectNoError(err)
+
+				cleanupSteps = append(cleanupSteps, func() {
+					// The purpose of this block is to prevent physical snapshotContent leaks.
+					// We must update the SnapshotContent to have Delete Deletion policy,
+					// or else the physical snapshot content will be leaked.
+					ginkgo.By("get the latest copy of volume snapshot content")
+					snapshotContent, err := dc.Resource(SnapshotContentGVR).Get(context.TODO(), vscontent.GetName(), metav1.GetOptions{})
+					framework.ExpectNoError(err)
+
+					ginkgo.By("updating the the SnapshotContent to have Delete Deletion policy")
+					snapshotContent.Object["spec"].(map[string]interface{})["deletionPolicy"] = "Delete"
+					vscontent, err = dc.Resource(SnapshotContentGVR).Update(context.TODO(), snapshotContent, metav1.UpdateOptions{})
+					framework.ExpectNoError(err)
+
+					ginkgo.By("manually deleting the SnapshotContent")
+					err = dc.Resource(SnapshotContentGVR).Delete(context.TODO(), snapshotContent.GetName(), metav1.DeleteOptions{})
+					if err != nil && !apierrors.IsNotFound(err) {
+						framework.Failf("Error deleting snapshot content %q. Error: %v", snapshotContent.GetName(), err)
+					}
+
+					ginkgo.By("checking the SnapshotContent has been deleted")
+					err = utils.WaitForGVRDeletion(dc, SnapshotContentGVR, vscontent.GetName(), framework.Poll, framework.SnapshotDeleteTimeout)
+					framework.ExpectNoError(err)
+				})
+
+				// Validate volume snapshot content is not removed
+				DeleteAndWaitSnapshot(dc, vs.GetNamespace(), vs.GetName(), framework.Poll, framework.SnapshotDeleteTimeout)
+				err = utils.WaitForGVRDeletion(dc, SnapshotContentGVR, vscontent.GetName(), 1*time.Second /* poll */, 30*time.Second /* timeout */)
+				framework.ExpectError(err) // should fail deletion check
+			})
+
+			ginkgo.It("should create objects with default", func() {
```

Comment:
what do you mean by "with default" here? Isn't this really just checking that the SnapshotContent and such was created correctly/

Reply:
This is actually the wording from the commit before the previous commit. I interpret it to mean as creating objects with default field values. The intention is to just check the creation is done correctly. I think a better wording may be "should create snapshot objects"